spec/factories.rb

```diff
@@ -1,2 +1,2 @@
-FactoryBot.define do
+FactoryBot.define do  
```

Comment:
There is an extra space after the `do`. Not sure what the rubocop rule is to check that